CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-07-30
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* For AKS clusters built at version v1.27+ and enable KMS, KMS v2 is configured by default. However, for clusters with KMS enabled at versions below v1.27, upgrading to v1.27 will be blocked. To upgrade, follow the steps outlined in this [documentation](https://learn.microsoft.com/azure/aks/use-kms-etcd-encryption#migration-to-kms-v2) for migrating from KMS v1 to v2, and then proceed with upgrading the cluster to version v1.27.
+* The pod security policy feature was deprecated on 1st August 2023 and removed since AKS version 1.25. We recommend you migrate to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) to stay within Azure support.
+
+
+### Release notes
+
+* Features
+  * The [AKS Vscode extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) now supports [cluster creation](https://azure.github.io/vscode-aks-tools/features/show-properties-azureportal-start-stop.html).
+  * Kubernetes version 1.27 is now Generally Available (GA).
+
+* Behavioral changes
+  * Kubernetes version 1.24 is now deprecated.
+  * The nodereadiness-retry-interval has changed from 10s to 5s.
+  * The nodereadiness-retry-max-count has changed from 5 to 12.
+  * Enable bpf-lb-sock-hostns-only option in Azure CNI Powered by Cilium for compatibility with Istio and other service meshes.
```

Comment:
there's another RP side change required to get rid of the blocking logic we have for Istio and Cilium. that's gonna happen next week. so let's remove this line this week. we will cover Istio + cilium enable next week after that RP side change is done